src/MarkovKernels.jl

```diff
@@ -55,6 +57,7 @@ include("distributions/particle_system.jl")
 include("distributions/plotting.jl")
 export AbstractNormal,
     Normal,
+    UvNormal,
```

Comment:
Should users actually use UvNormal or is that just a convenience thing to restrict function input types?

Reply:
Then this should not be exported imo

Reply:
I fixed.